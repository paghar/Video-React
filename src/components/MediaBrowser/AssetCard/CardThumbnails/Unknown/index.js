//@flow
import React from 'react';
import { UnknownFileThumbnail } from './index.style';
import { Generals } from 'theme/colors';
import { hexToRgb } from 'helpers';

type Props = {
  className?: string,
};

const UnknownThumbnail = ({ className }: Props) => {
  return (
    <UnknownFileThumbnail
      className={className}
      backgroundColor={hexToRgb(Generals.iii, 0.2)}
      activeBackgroundColor="transparent"
      icon="61787"
      iconSize="40px"
      iconColor={Generals.ii}
    />
  );
};

export default UnknownThumbnail;
