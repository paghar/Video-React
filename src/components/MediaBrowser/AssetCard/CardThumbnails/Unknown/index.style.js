//@flow
import styled from 'styled-components';
import Button from 'components/shared/Button';

export const UnknownFileThumbnail = styled(Button)`
  pointer-events: none;
  width: 100%;
  height: 100%;
  display: flex;
`;
