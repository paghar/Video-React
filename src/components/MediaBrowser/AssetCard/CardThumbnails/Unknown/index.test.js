//@flow
import React from 'react';
import renderer from 'react-test-renderer';
import Unknown from '.';
import cardImage from 'assets/image/Colleagues_looking_at_information_on_mobile_phone_screen_Videos_Mixkit.png';

describe('<AssetCardUnknownThumbnail />', () => {
  it('Unknown Thumbnail Snapshot test', () => {
    const tree = renderer.create(<Unknown />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
