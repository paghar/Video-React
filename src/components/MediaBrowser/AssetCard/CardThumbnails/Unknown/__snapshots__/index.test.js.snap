// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AssetCardUnknownThumbnail /> Unknown Thumbnail Snapshot test 1`] = `
.c2 {
  font-family: 'FontAwesome' !important;
  font-size: 40px;
  margin: 0px;
  padding: 0px;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  -moz-user-select: none;
  -khtml-user-select: none;
  -webkit-user-select: none;
  -ms-user-select: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.c3 {
  font-size: 40px;
  display: block;
  padding-left: 0px;
  -moz-user-select: none;
  -khtml-user-select: none;
  -webkit-user-select: none;
  -ms-user-select: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.c0 {
  direction: ltr;
  outline: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: none;
  background: none;
  background-color: rgba(0,0,15,0.2);
  -moz-user-select: none;
  -khtml-user-select: none;
  -webkit-user-select: none;
  -ms-user-select: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.c0:active {
  background-color: transparent;
}

.c1 {
  pointer-events: none;
  width: 100%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<button
  className="c0 c1"
>
  <span
    className="c2 c3"
    fontSize="40px"
  >
    ï…›
  </span>
</button>
`;
